### ISSUE TITLE
1.16.2

### ISSUE THREAD

**Issue by Jackis18** [iSach/UltraCosmetics#509]
### What steps will reproduce the issue?  
1.  Use spigot or Paper 1.16.2

### What was supposed to happen?

Plugin should startup.

### What happened? 

Plugin does not start,

22.08 13:14:40 [Server] INFO [UltraCosmetics] Enabling UltraCosmetics v2.5.8
22.08 13:14:40 [Server] INFO 08/22/2020 - UltraCosmetics [INFO]: This NMS version isn't supported. (v1_16_R2)!
22.08 13:14:40 [Server] INFO ----------------------------
22.08 13:14:40 [Server] INFO ULTRACOSMETICS CAN ONLY RUN ON 1.8 through 1.16.1!
22.08 13:14:40 [Server] INFO ----------------------------
22.08 13:14:40 [Server] INFO [UltraCosmetics] Disabling UltraCosmetics v2.5.8

### What version of UltraCosmetics are you using?  

2.5.8

### What Spigot version are you using? Paste the output of /version below.  

22.08 13:18:54 [Server] INFO JackDotMC issued server command: /version
22.08 13:18:59 [Server] INFO This server is running Paper version git-Paper-"dccaf59a0" (MC: 1.16.2) (Implementing API version 1.16.2-R0.1-SNAPSHOT)
22.08 13:18:59 [Server] INFO Previous version: git-Paper-135 (MC: 1.16.1)
22.08 13:18:59 [Server] INFO Error obtaining version information

### What plugins do you have installed? Paste the output of /plugins below.  

N/A

### Are any errors related to UltraCosmetics in your console or logs? If so, paste below.  
```
22.08 13:14:40 [Server] INFO [UltraCosmetics] Enabling UltraCosmetics v2.5.8
22.08 13:14:40 [Server] INFO 08/22/2020 - UltraCosmetics [INFO]: This NMS version isn't supported. (v1_16_R2)!
22.08 13:14:40 [Server] INFO ----------------------------
22.08 13:14:40 [Server] INFO ULTRACOSMETICS CAN ONLY RUN ON 1.8 through 1.16.1!
22.08 13:14:40 [Server] INFO ----------------------------
22.08 13:14:40 [Server] INFO [UltraCosmetics] Disabling UltraCosmetics v2.5.8
```

**Comment by SilkPenny** [comment]
Same issue here.  Is there any update on this?

**Comment by Permanently** [comment]
> Same issue here. Is there any update on this?
Seems 2.5.9 will be the fix for you.

**Comment by datatags** [comment]
Fixed in #511
